auto_round/special_model_handler.py

```diff
@@ -41,6 +41,10 @@ def _get_deepseek_vl2_multimodal_block(model, quant_vision=False):
     "deepseek_vl_v2": _get_deepseek_vl2_multimodal_block
 }
 
+def _deepseek_vl2_forward(model, *args, **kwargs):
+    ret = model.prepare_inputs_embeds(*args, **kwargs)
+    return model.language(inputs_embeds=ret, attention_mask=kwargs["attention_mask"])
```

Comment:
If attention_mask is not always required, consider adding a check; otherwise, make it an explicit parameter.